tooling/build/scripts/publisher.sh

```diff
@@ -70,7 +70,6 @@ start_time=$(date +%s)
 cd tooling/build/scripts/publishing
 echo $(pwd)
 npm ci
-npm install ts-node -g
```

Comment:
we don't need ts-node at build time?

Reply:
we shouldn't specify dependencies like this - we have a `package.json`. doing this makes it confusing when trying to test locally (as i found out). 

what happened is that:
1. i specified `tsx` as a dependency (faster, respects `tsconfig.paths` so we can share types from db -> build, which is important to make sure our build step works properly)
2. updated the `npm run start` command to use `tsx` - which is now a dependency

Reply:
oops missed the addition of tsx in package. Okay lgtm